app/assets/stylesheets/prism.scss

```diff
@@ -0,0 +1,159 @@
+
```

Comment:
You can pull in govuk-elements-sass as a dependency though and that means that when you update elements you update with any improvements that might have been made to the sass within elements.